sirepo/package_data/static/json/srw-schema.json

```diff
@@ -674,7 +674,7 @@
             "numWorkers": ["Number of Workers", "Integer", 1],
             "randomSeed": ["Random Seed", "RandomSeed", "", "Explicit seed for random numbers in the range [{{ SIREPO.APP_SCHEMA.model.exportRsOpt.randomSeed[SIREPO.INFO_INDEX_MIN] }}, {{ SIREPO.APP_SCHEMA.model.exportRsOpt.randomSeed[SIREPO.INFO_INDEX_MAX] }}). Leave empty to allow server to decide the seed.", 0, 4294967296],
             "scanType": ["Scan Type", "ScanType", "random", "Grid samples each parameter uniformly. Random takes a random sample for each parameter."],
-            "totalSamples": ["Total Samples", "Integer", 100, "Grid: &lt;number of samples&gt;^&lt;number of params&gt;. Random: &lt;number of samples&gt;", 1],
+            "totalSamples": ["Total Samples", "Integer", 100, "Grid: &lt;number of samples&gt;^&lt;number of params&gt;. Random: &lt;number of samples&gt;", 0],
```

Comment:
We should change the default to 0 - new simulations see 100 and the sim panel shows up. Maybe also run `scope.updateTotalSamples()` and save quietly when the rsOptElements directive first loads.

Reply:
I update totalSamples default to 0 now